axon-framework/events/event-bus-and-event-store.md

```diff
@@ -393,14 +393,23 @@ If no explicit `eventSerializer` is configured, events are serialized using the
 
 ## Distributing Events
 
-The use of an `EventStore` provides a simple way of distributing events: by sharing the data source.
-This is the approach taken by [Axon Server](../../axon-server-introduction.md), for example.
-
-Such an approach will also suffice whenever the [`EmbeddedEventStore`](#embedded-event-store) is used.
-Doing so requires the correct configuration of each application instance to point to the same data source.
-Listing _all_ possible data sources is out of the scope of this guide however.
-The documentation for the chosen source should thus be read for specifics on this.
-
-Alternatively, you can choose other components to distribute events.
-Axon provides a couple of these as [extension modules](../../extensions) for example [Spring AMQP](../../extensions/spring-amqp.md) or [Kafka](../../extensions/kafka.md).
-Note that neither of these are intended as an [Event Store](#event-store), but **only** as a message broker.
\ No newline at end of file
+To distribute events between applications, it is paramount to know whether the applications belong to the same [bounded context](../../architecture-overview/ddd-cqrs-concepts.md#bounded-context).
+If they do, they're allowed to understand each other's language.
+In other words, they should be free to understand the events altogether.
+
+As such, we can share the `EventStore`'s data source between these applications typically suffices.
+We may thus achieve distribution by utilizing the source itself.
+You can use both the [`EmbeddedEventStore`](#embedded-event-store) and [Axon Server](../../axon-server-introduction.md) for this.
+The former would require the applications to point to the same data source, whereas the latter would require the applications to partake in the same context.
+
+However, sharing the event store is not recommended whenever the applications do not belong to the same context.
+Instead, we should protect the boundary of the contexts, both from the event's perspective and the storage solution.
+
+To distribute events between bounded contexts, you can use Axon Server's [multi-context](../../axon-server/administration/multi-context.md) solution, for example.
+The former requires application registration to specific contexts, firstly.
+Secondly, you can open a stream to another context through the `AxonServerEventStore#createStreamableMessageSourceForContext(String)` operation.
+With this source in hand, you can configure a [Streaming Processor](event-processors/streaming.md) to start reading from it.
+
+Alternatively, you can choose a message broker to distribute events.
```

Comment:
It is not super clear to me we moved from Event Storage to message distribution... when I get to this sentence, it was not clear to me that AS also supports message distribution ootb

Reply:
Honestly, I am not entirely clear on how to change this comment into a workable adjustment for this pull request. The section is called "Distributing Events", which within a context is simply achieved by sharing the source.

As sharing a data source is not recommended when the applications don't belong to one another, a different approach is needed.
This is where we have roughly two options:
1. Let Axon Server allow opening a stream
2. Add a third-party piece of infrastructure to distribute events

Reply:
I will think a bit more about it and get back to you... or we can just merge this one and discuss further improvements on another issue and future PR!